site/en/docs/devtools/recorder/index.md

```diff
@@ -160,6 +168,42 @@ There are options to add and remove steps too. This is useful if you want to add
    {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/zeK91O21jvc5OD9HuovR.png", alt="The new step for coffee checkout has been updated with the aforementioned details.", width="800", height="663" %}
 6. [Replay](#replay) the flow now to see the changes.
 
+## Share user flows {: #share-flows }
+
+You can export and import user flows. This is useful for bug reporting because you can share an exact record of the steps that reproduce a bug.
+
+### Export a user flow {: #export-flows }
+
+To export a user flow:
+
+1. Open the user flow you want to export.
+2. Click on the **Export**<span class="material-icons">file_download</span> button at the top of the **Recorder** panel.
+    {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/LJJqpZ0vOEdp6rdDLXmP.png", alt="Download format options", width="800", height="490" %}
+ 1. Select one of the following formats from the dropdown list:
+    - **Export as a JSON file**. Download the recording as a JSON file.
+    - **Export as a @puppeteer/replay script**. Download the recording as a [Puppeteer Replay](https://github.com/puppeteer/replay) script.
+    - **Export as a Puppeteer script**. Download the recording as a [Puppeteer](https://pptr.dev/) script.
+1. Save the file.
+
+{% Aside 'gotchas' %}
+The [@puppeteer/replay](https://github.com/puppeteer/replay) is a library built on top of [Puppeteer](https://pptr.dev/). It is also a command line tool, so you can replay JSON files with it too.
```

Comment:
@mathiasbynens @OrKoN the mention is here. PTAL. Once the doc is in, we will update the WNDT 101 link to point to this section

Reply:
Ack, thanks!

I think what’s missing from this explanation is that it can be used not just to replay, but also to “stringify” user flow JSON files, i.e. convert them to something else. We should also mention that tooling authors can build their own replay/stringify integrations on top of this library. @OrKoN